ql/src/java/org/apache/hadoop/hive/ql/io/orc/encoded/EncodedReaderImpl.java

```diff
@@ -282,6 +280,56 @@ public String toString() {
     }
   }
 
+  public static boolean[] findPresentStreamsByColumn(
```

Comment:
Can we add javadoc for these public static utility methods? If they are used only in this class, should we change their visibility?

Reply:
Sure, done